controls/cis_sle12.yml

```diff
@@ -1038,7 +1040,9 @@ controls:
     levels:
     - l1_server
     - l1_workstation
-    status: manual 
+    status: automated
```

Comment:
instead of having this as "automated" should it then have something like "partial" and should you add "notes" to say that it doesn't contain oval and remediation because it is a manual item?

Reply:
Yes, maybe. @marcusburghardt do you have anything against it?

Reply:
I agree that keeping its status as `manual` would be better in case CIS profiles.
It will also ensure the accuracy of the metrics when using the `controleval.py` tool. e.g.:
`utils/controleval.py stats -p sle15 -i cis_sle15 -l l1_server`